.claude/commands/shared/research-patterns.yml

```
@@ -0,0 +1,463 @@
+    1: "Detect library reference in code/request"
+    2: "Check if already in package.json/requirements.txt"
+    3: "C7 resolve-library-id → get-docs with topic"
+    4: "If C7 fails → WebSearch '[library] official documentation'"
+    5: "Extract: Installation|Basic usage|Common patterns|Error handling|Best practices"
+    6: "Cache results for session with confidence score"
+    7: "Cite sources in implementation"
+  Blocking_Conditions:
+    - "External library detected without research"
+    - "Confidence score below 90%"
+    - "No official documentation found"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)